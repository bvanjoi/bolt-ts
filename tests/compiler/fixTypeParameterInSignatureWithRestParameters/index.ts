// From `github.com/microsoft/TypeScript/blob/v5.9.2/tests/cases/compiler/fixTypeParameterInSignatureWithRestParameters.ts`, Apache-2.0 License

function bar<T>(item1: T, item2: T) { }
bar(1, ""); // Should be ok
//~^ ERROR: Argument of type 'string' is not assignable to parameter of type 'number'.