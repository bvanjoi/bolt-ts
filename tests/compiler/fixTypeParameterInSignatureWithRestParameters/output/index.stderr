  × Argument of type 'string' is not assignable to parameter of type 'number'.
   ╭─[../../tests/compiler/fixTypeParameterInSignatureWithRestParameters/index.ts:4:8]
 4 │ ""
   · ──
   ╰────
